docs/text/rolling.rst

```diff
@@ -21,6 +21,11 @@ the last 10 steps.
 A rolling mechanism creates such time series for every time step by creating sub time series of the sensor data of the
 last 10 time steps.
 
+<!---
+The above scenario cannot be solved by tsfresh, can it?
```

Comment:
yes, it can. 

the `maximum_number_of_timeshifts` parameter of the `roll_time_series` controls how many ticks you look into the past (here 10)